.github/workflows/new-plugin.yml

```diff
@@ -0,0 +1,96 @@
+name: new-plugin
+on:
+  push:
+    branch:
+      - "main"
+    paths:
+      - "plugins/**"
+
+jobs:
+  job1:
+    name: TODO name the job
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+    if: github.repository == 'nix-community/nixvim'
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ssh-key: ${{ secrets.CI_UPDATE_SSH_KEY }}
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v26
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get plugins info
+        run: |
+          OLD="$(
+            nix eval 'github:${{ github.repository }}/${{ github.event.before }}#nixvimConfiguration.options.plugins' \
+              --apply 'builtins.attrNames' \
+              --json
+          )"
+          NEW="$(
+            nix eval '.#nixvimConfiguration.options.plugins' \
+              --apply 'builtins.attrNames' \
+              --json
+          )"
+          # TODO: compare OLD and NEW
+
+      - name: Get PR info
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          if JSON="$(
+            gh api  \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              /repos/${{ github.repository }}/commits/${{ github.sha }}/pulls
+          )"
+          then
+            # TODO: use PR info
+          else
+            # TODO: handle error getting PR info
+          fi
+
+      - name: Send matrix message
+        uses: lkiesow/matrix-notification@v1
+        with:
+          # The Matrix home server to send the message to
+          # Default: matrix.org
+          server: matrix.org
+          # TODO: token
+          token: ...
```

Comment:
Should be able to have @GaetanLepage set up a repo environment variable for this that we can reference

Reply:
Yeah. It's documented upstream [here](https://github.com/lkiesow/matrix-notification?tab=readme-ov-file#matrix-token). I guess we'd want a dedicated "bot" user?